user-service/src/test/java/rs/edu/raf/banka1/services/PaymentServiceImplTest.java

```diff
@@ -0,0 +1,322 @@
+package rs.edu.raf.banka1.services;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.any;
+import static org.mockito.Mockito.anyString;
+import static org.mockito.Mockito.eq;
+import static org.mockito.Mockito.never;
+import org.mockito.junit.jupiter.MockitoExtension;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import rs.edu.raf.banka1.mapper.PaymentMapper;
+import rs.edu.raf.banka1.dtos.PaymentDto;
+import rs.edu.raf.banka1.model.*;
+import rs.edu.raf.banka1.repositories.BankAccountRepository;
+import rs.edu.raf.banka1.repositories.CustomerRepository;
+import rs.edu.raf.banka1.repositories.PaymentRepository;
+import rs.edu.raf.banka1.requests.CreatePaymentRequest;
+import rs.edu.raf.banka1.services.implementations.PaymentServiceImpl;
+import rs.edu.raf.banka1.utils.JwtUtil;
+
+import java.util.Optional;
+import java.util.List;
+import java.util.ArrayList;
+import static junit.framework.TestCase.assertNotNull;
+import static org.mockito.Mockito.when;
+
+
+@ExtendWith(MockitoExtension.class)
```

Comment:
dodaj @mockitosettings lenient ubuduce, jer ume da zeza ako se ne stavi (npr ako se ne iskoristi neki mock, padne test za dzabe)

Reply:
a oke,videla sam da negde to ima a negde nema